src/components/device/ha-device-action-picker.ts

```diff
@@ -7,15 +8,27 @@ import {
 import "../ha-paper-dropdown-menu";
 import { HaDeviceAutomationPicker } from "./ha-device-automation-picker";
 
-@customElement("ha-device-action-picker")
-class HaDeviceActionPicker extends HaDeviceAutomationPicker<DeviceAction> {
-  protected NO_AUTOMATION_TEXT = "No actions";
```

Comment:
Make this a getter that returns `hass.localize("ui.panel.config.devices.automation.actions.no_actions");`

Reply:
That solution feels a lot cleaner. Shows, that I am quite new to TS, since I wasn't aware of that concept yet. Thanks for the hint!